sktime/forecasting/compose/_ensemble.py

```diff
@@ -73,6 +74,29 @@ def update(self, y, X=None, update_params=True):
         return self
 
     def _predict(self, fh, X=None, return_pred_int=False, alpha=DEFAULT_ALPHA):
+        """return the predicted reduction
+
+        Parameters
+        ----------
+        fh : int, list or np.array, optional (default=None)
+        X : pd.DataFrame
+        return_pred_int : boolean, optional (default=False)
+        alpha : fh : float, (default=DEFAULT_ALPHA)
+
+        Returns
+        -------
+        DataFrame : aggregate dataframe reduction.
+        """
         if return_pred_int:
             raise NotImplementedError()
+        if self.agg not in ("median", "mean", "min", "max"):
```

Comment:
I would write something like this
```suggestion
        valid_agg_funcs = ("median", "mean", "min", "max")
        if self.agg not in valid_agg_funcs:
```
You can then reuse `valid_agg_funcs` in the error message.